scratch/evaluate_gaming_traffic.cc

```diff
@@ -25,30 +25,85 @@
 #include "ns3/internet-module.h"
```

Comment:
this should probably be in examples rather than scratch

Reply:
You mean the `evaluate_gaming_traffic.cc` file?

Reply:
yes

Reply:
@MatteoDrago @tommyazz ?

Reply:
I think we also should move the `mixture-random-variable` and `three-lognormal-random-variable` somewhere else, possible to the `random-variable-stream`.

Reply:
> @MatteoDrago @tommyazz ?

Yes, I think it would be better to move it in examples. However, with the actual configure we are forced to disable examples. So, you will not be able to run it if you move it.

Reply:
> I think we also should move the `mixture-random-variable` and `three-lognormal-random-variable` somewhere else, possible to the `random-variable-stream`.

`random-variable-stream`: (if I'm not wrong) this is a unique file that contains the definition of all ns-3 random variables. Maybe it's good to keep "our" rvs in separated files.

Reply:
> Yes, I think it would be better to move it in examples. However, with the actual configure we are forced to disable examples. So, you will not be able to run it if you move it.

you're right, i forgot about that. we can move it as a last thing before the merge